crates/bevy_pbr/src/ssr/mod.rs

```diff
@@ -190,50 +190,41 @@ impl Plugin for ScreenSpaceReflectionsPlugin {
 
         render_app
             .init_resource::<ScreenSpaceReflectionsBuffer>()
+            .init_resource::<SpecializedRenderPipelines<ScreenSpaceReflectionsPipeline>>()
+            .add_systems(
+                RenderStartup,
+                (
+                    init_screen_space_reflections_pipeline,
+                    add_screen_space_reflections_render_graph_edges,
+                ),
+            )
             .add_systems(Render, prepare_ssr_pipelines.in_set(RenderSystems::Prepare))
             .add_systems(
                 Render,
                 prepare_ssr_settings.in_set(RenderSystems::PrepareResources),
             )
+            // Note: we add this node here but then we add edges in
+            // `add_screen_space_reflections_render_graph_edges`.
             .add_render_graph_node::<ViewNodeRunner<ScreenSpaceReflectionsNode>>(
```

Comment:
Is there any reason why you didn't move this? Is it just to keep behaviour as close as possible? Oh, could it break something later if the node isn't added in build?

Reply:
Mostly to try to keep the diff small, and also because initializing ViewNodeRunner requires &mut World, so I was hoping to avoid requiring &mut World in the system.

In fact I was considering going the other way and instead moving the one required graph edge here, and then only do the conditional edge in the system. But alas I decided to try to keep the edges together at least.

Reply:
Okay, keeping the diff small is fine by me. We'll likely want to do a cleanup pass after 0.18 but it's probably good to keep everything as close as what it was before for now.